operators/pkg/instctrl/cloudinit.go

```diff
@@ -44,14 +44,18 @@ const (
 // based on the information retrieved for the tenant object and its associated WebDav credentials.
 func (r *InstanceReconciler) EnforceCloudInitSecret(ctx context.Context) error {
 	log := ctrl.LoggerFrom(ctx)
-
-	// Retrieve the WebDav credentials.
-	user, password, err := r.GetWebDavCredentials(ctx)
-	if err != nil {
-		log.Error(err, "unable to get webdav credentials")
-		return err
+	env := clctx.EnvironmentFrom(ctx)
+	var server string
+	var path string
+	var err error
+
+	if env.MountPathEnabled {
+		_, server, path, err = r.GetNfsSpecs(ctx)
+		if err != nil {
```

Comment:
IIRC, you should be able to do something alike, also removing `var err error`:
```suggestion
		if _, server, path, err := r.GetNfsSpecs(ctx); err != nil {
```

Reply:
Ok, removed `var err error`